Source/Core/Core/HW/SI_GCAdapter.cpp

```diff
@@ -398,6 +400,23 @@ void Input(int chan, GCPadStatus* pad)
 	}
 }
 
+void ResetRumble()
+{
+	if (!SConfig::GetInstance().m_GameCubeAdapter)
+		return;
+	if (s_handle == nullptr || !s_detected)
+		return;
+
+	std::fill_n(s_controller_rumble, 4, 0);
```

Comment:
this can just be a regular fill with

```cpp
std::fill(std::begin(s_controller_rumble), std::end(s_controller_rumble), 0);
```

Gets rid of hardcoding a size.